02-peer-protocol.md

```diff
@@ -519,6 +521,9 @@ In general, we need to complete one side of the exchange before dealing with the
 Fulfilling an HTLC is different: knowledge of the preimage is by definition irrevocable, 
 so we should fulfill the incoming HTLC as soon as we can to reduce latency.
 
+An HTLC with an extremely long expiry is a denial-of-service vector,
+so is not allowed: the exact value of reasonable is currently unclear
```

Comment:
"so it is"

Reply:
It's actually correct in British English, but slightly old-fashioned/formal. I often get picked up on it by Americans, for whom it's not idiomatic.  Will fix...